### PR TITLE
Documenting Additional Downward APIs

### PR DESCRIPTION
**Pull request by brandoncole**
There are a couple awesome enhancements that would benefit from being documented here:

1. `spec.nodeName` for obtaining the host name
2. `spec.serviceAccountName` for obtaining the name of the service account the pod is running under

Right now these enhancements are documented only in a couple of issues and pull requests:

1. kubernetes/kubernetes#27880
2. kubernetes/kubernetes#24657
3. kubernetes/kubernetes#21317 (I suggested this change here and it got a few thumbs up)

I have verified the functionality in the latest codebase:

1. https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/conversion.go#L200

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1885)

### CODE REVIEW

**Comment by brandoncole** [comment]
@chrislovecnm @jroberts235 @gg7 @kris-nova  - FYI

**Comment by jaredbhatti** [comment]
@pmorie  Can you review this?

**Comment by mikedanese** [comment]
Tech lgtm, please squash.

**Comment by chrislovecnm** [comment]
@bcole can u squash the commit?

File: docs/user-guide/downward-api/index.md [hunk 1]
```diff
@@ -22,13 +22,15 @@ the Pod's name, for example, and inject it into this well-known variable.
...
+*   The node's name 
+*   The pod's name
 *   The pod's namespace
 *   The pod's IP
+*   The pod's service account name 
 *   A container's cpu limit
 *   A container's cpu request
 *   A container's memory limit
-*   A container's memory request
+*   A container's memory request
```
Comment (chrislovecnm) [inline docs/user-guide/downward-api/index.md]: Spaces ;)

**Comment by brandoncole** [comment]
Thanks @chrislovecnm for spotting the spaces and @jaredbhatti @mikedanese for the review.  It's squashed and spaces removed.  Happy New Year everyone.

**Comment by chrislovecnm** [comment]
/LGTM

---

Reviewed 1 of 2 files at r1, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/1885#-:-Ka-qsmcQj8tvQeTnxiW:b-8ld9tu)*

**Comment by chrislovecnm** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/1885#-:-Ka-qy8vkbXiCrHgH24k:bnfp4nl)*

**Comment by brandoncole** [comment]
Just updated the branch.  The Travis build seemed to have a hiccup earlier today.  If there's anything I can do to help move this along please let me know @chrislovecnm @mikedanese

**Comment by chrislovecnm** [comment]
Travis failed.

**Comment by chenopis** [comment]
@brandoncole @chrislovecnm I updated the branch, which fixed the Travis check. That was broken because of an unrelated issue that has since been resolved. Cheers.

**Comment by brandoncole** [comment]
Thanks everyone - much appreciated!

**Pull request merged**